addons/purchase_requisition/views/purchase_requisition_views.xml

```diff
@@ -103,21 +33,15 @@
             <field name="company_id" invisible="1"/>
             <field name="currency_id" invisible="1"/>
             <header>
-                <button name="%(action_purchase_requisition_to_so)d" type="action"
-                    string="New Quotation"
-                    context="{'default_currency_id': currency_id, 'default_user_id': user_id}"
-                    invisible="state != 'open'"/>
                 <button name="%(action_purchase_requisition_to_so)d" type="action"
                     string="New Quotation" class="btn-primary"
                     context="{'default_currency_id': currency_id, 'default_user_id': user_id}"
-                    invisible="state not in ('in_progress', 'ongoing')"/>
+                    invisible="state != 'ongoing'"/>
                 <button name="action_in_progress" invisible="state != 'draft'" string="Confirm" type="object" class="btn-primary"/>
-                <button name="action_open" invisible="state != 'in_progress'" string="Validate" type="object" class="btn-primary"/>
-                <button name="action_done" invisible="state not in ('open', 'ongoing')" string="Close" type="object" class="btn-primary"/>
+                <button name="action_done" invisible="state != 'ongoing' or requisition_type == 'purchase_template'" string="Close" type="object" class="btn-primary"/>
                 <button name="action_draft" invisible="state != 'cancel'" string="Reset to Draft" type="object"/>
-                <button name="action_cancel" invisible="state not in ('draft', 'in_progress', 'ongoing')" string="Cancel" type="object"/>
-                <field name="state" widget="statusbar" statusbar_visible="draft,in_progress,open,done" invisible="is_quantity_copy == 'none'"/>
-                <field name="state_blanket_order" widget="statusbar" statusbar_visible="draft,ongoing,done" invisible="is_quantity_copy != 'none'"/>
+                <button name="action_cancel" invisible="state not in ('draft', 'ongoing')" string="Cancel" type="object"/>
+                <field name="state" widget="statusbar" statusbar_visible="draft,ongoing,done" invisible="requisition_type == 'purchase_template'"/>
```

Comment:
talked to Thomas directly: allow archiving purchase agreements, this includes for Blanket Orders (there's no reason why we shouldn't be able to archive them, it's probably legacy code that they aren't archive-able). This will fix this issue